--- shell.nix	2019-04-10 21:54:57.000000000 +0200
+++ shell_new.nix	2019-04-10 21:54:57.000000000 +0200
@@ -1,4 +1,4 @@
-{ nixpkgs ? import <nixpkgs> {}, compiler ? "default", doBenchmark ? false }:
+{ nixpkgs ? import <nixpkgs> { config = import ./docker-repl-config.nix; }, compiler ? "default", doBenchmark ? false }:
 
 let
 
@@ -41,6 +41,12 @@
         ];
         preConfigure = "hpack";
         license = stdenv.lib.licenses.gpl2;
+        buildDepends = [
+            pkgs.cabal-install
+            pkgs.wget
+            pkgs.openssh
+            pkgs.mysql
+        ];
       };
 
   haskellPackages = if compiler == "default"
